### PR TITLE
Count Down Timer

### PR DESCRIPTION
**Pull request by Phildeoner**
Please Kindly Review the project and merge.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// Set Count Down date
+let countDownDate = new Date("Jan 1, 2024 00:00:00").getTime();
+
+// Update count down timer every 1 second
```
Comment (DanSam5K) [inline script.js]: - [Optional] It is often considered a good practice to remove comments before merging into the main branch so you might want to remove inline comment then merge.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    // If the count down is finished, write some text
+    if (interval < 0) {
+        clearInterval(timeUpdate);
+        document.getElementById("days").innerHTML = 000;
+        document.getElementById("hours").innerHTML = 00;
+        document.getElementById("mins").innerHTML = 00;
+        document.getElementById("secs").innerHTML = 00;
+        document.getElementById("timeUp").innerHTML = "🎇HAPPY NEW YEAR🎇";
+    }
```
Comment (DanSam5K) [inline script.js]: - [Optional] Seeing that we have repeated document.getElementById we might want to research to see if we can [DRY](https://dev.to/richardwynn/dry-principle-in-100-seconds-2l8c) up the code what do you think?

**Review by DanSam5K — APPROVED** [review]
### Hi @phildeone, ✌️

- You can check the optional suggestion, see if you can change something🌿🙌

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

-

File: script.js (hunk 2 above)
Comment (Phildeoner) [reply script.js]: OK Sir thank you

File: script.js (hunk 1 above)
Comment (Phildeoner) [reply script.js]: OK Sir thank you

**Pull request merged**